docs/server-prometheus.md

```diff
@@ -9,27 +9,27 @@ The endpoint is available at the applications admin port at `/metrics/prometheus
-| **`io_dropwizard_jetty_`**                  |                                                          | Multiple metrics from the embedded Jetty server                                              | Bridged from Dropwizard                   |               |
-| **`io_dropwizard_db_`**                     |                                                          | Multiple metrics from the database if a database is used                                     | Bridged from Dropwizard                   |               |
-| **`healthcheck_status`**                    | _`name`_                                                 | Metrics that represent the state of the health checks                                        | `HealthCheckMetricsCollector`             |               | 
+| Metric name                                 | Labels                  | Description                                                                                                 | Source                                    | Deprecated    |
+|---------------------------------------------|-------------------------|-------------------------------------------------------------------------------------------------------------|-------------------------------------------|---------------|
+| **`http_request_duration_seconds`**         |                         | Tracks the time needed to handle a request                                                                  | `RequestDurationFilter`                   |               | 
+|                                             | _`implementing_method`_ | The name of the method that handled the request.                                                            | Request Context                           |               |
+|                                             | _`http_method`_         | The HTTP method the client used for the request.                                                            | Request Context                           |               |
+|                                             | _`resource_path`_       | The mapped path of the request with path param placeholders.                                                | Request Context                           |               |
+|                                             | _`status_code`_         | The HTTP status code sent with the response.                                                                | Response Context                          |               |
+|                                             | _`consumer_name`_       | Name of the consumer that started the request.                                                              | Request Context Property `Consumer-Name`* |               |
+| **`kafka_consumer_records_lag`**            |                         | **DEPRECATED** See [Kafka Documentation](https://kafka.apache.org/documentation/#consumer_fetch_monitoring) | Bridged from Kafka                        |               | 
```

Comment:
Could we mention the Micrometer alternative for the deprecated metrics?

Reply:
Added the replacements for `kafka_consumer_records_lag` and `kafka_producer_topic_message_total`

However there is no equivalent for `kafka_consumer_topic_message_duration` histogram since this is not an internal Kafka metric. What can be used is the `kafka_consumer_fetch_manager_records_consumed_rate` gauge which is inverse to above message duration metric.